cvat-ui/src/actions/tasks-actions.ts

```diff
@@ -213,6 +219,55 @@ export function loadAnnotationsAsync(
     };
 }
 
+function importTask(): AnyAction {
+    const action = {
+        type: TasksActionTypes.IMPORT_TASK,
+        payload: {},
+        importing: true,
+    };
+
+    return action;
+}
+
+function importTaskSuccess(taskId: number): AnyAction {
+    const action = {
+        type: TasksActionTypes.IMPORT_TASK_SUCCESS,
+        payload: {
+            taskId,
+            importing: false,
+        },
+    };
+
+    return action;
+}
+
+function importTaskFailed(error: any): AnyAction {
+    const action = {
+        type: TasksActionTypes.IMPORT_TASK_FAILED,
+        payload: {
+            error,
+            importing: false,
```

Comment:
literal values should not be a part of payload because we can specify them in reducer based on action type